### DISCUSSION TITLE
Scrollbar for large lists

### DISCUSSION THREAD

**Discussion by Chris78c** [music-assistant/support#2554] [Feature requests and ideas]
I have on the NAS, then managed by home assistant as a Filesystem (local file), More than 10000 music files, 2000 artists and albums I'd like to scroll through with a bar.  Of course in the **Mobile App**.

If the bar then had a pop-up with the initial to understand alphabetically where we are on the list, it would be great. Thank you

(In the video I just want to show that while scrolling there is no scroll bar.) 

https://github.com/music-assistant/hass-music-assistant/assets/165342443/e0c301ba-03a0-484b-9a26-78c6512c5b74

**Comment by guussprokkereef** [comment]
This would be really helpful especially if an index is added to jump immediately to a specific letter to start browsing artists songs etc starting with that letter.

**Reply by vingerha** [reply]
I just created a new request with this very same demand.

**Reply by perlchamp** [reply]
Where ?

**Reply by vingerha** [reply]
https://github.com/orgs/music-assistant/discussions/3646
noting this is only covering the sub-discussion and not the topic of scrollbar
and it hasn't received much attention too so likely I am one of few that want it :)
I too have thousands and thousands of albums/tracks/etc. so 'contains'  is helpful too but more than often (with me) it is not the best

**Reply by perlchamp** [reply]
every library-website i know uses these headline for better navigation. MA not. It's a shame imo.
MusicBrainz' influence on MA is also noteworthy, and of course the fact that to create playlists, on the remote shares write permissions must be granted so that MusicBrainz can modify the files (inserting additional tags and ids into music files - that's a no-go at all!). Nobody modify my files, nobody!

**Reply by vingerha** [reply]
Not yet  I would say. There is only so much time they can spend on things and this then becomes low prio seeing that only you and me ... 2 people ... wanting it vs a lot of people not being really interested.

**Reply by OzGav** [reply]
> the fact that to create playlists, on the remote shares write permissions must be granted so that MusicBrainz can modify the files

Where did you find this “fact”?

**Comment by perlchamp** [comment]
Yes, please

**Reply by fakethinkpad85** [reply]
+1

**Comment by kekoehn-meep-meep** [comment]
+1 for any 'jump to' 

Absolutely a necessity for large libraries. Like chris78c, I have thousands of artists, albums, and tracks.  Scrolling through artists to "K" is next to impossible as it currently stands.
